spec.html

```diff
@@ -30726,9 +30784,30 @@ <h1>String.prototype [ @@iterator ] ( )</h1>
         <emu-alg>
           1. Let _O_ be ? RequireObjectCoercible(*this* value).
           1. Let _S_ be ? ToString(_O_).
-          1. Return CreateStringIterator(_S_).
+          1. Return ? Call(%StringIterator%, [[ _O_ ]]).
         </emu-alg>
         <p>The value of the *"name"* property of this function is *"[Symbol.iterator]"*.</p>
+
+        <emu-clause id="sec-string.prototype-@@iterator-stringiterator">
+          <h1>%StringIterator% ( _O_ )</h1>
+          <p><dfn>%StringIterator%</dfn> is a built-in generator function which</p>
```

Comment:
i don't think "which" works with the wording of the first two bullet points; is there another way we could segue into the bulleted list?